src/components/markdown-renderer/replace-components/markmap/markmap-frame.tsx

```diff
@@ -46,11 +46,15 @@ export const MarkmapFrame: React.FC<MarkmapFrameProps> = ({ code }) => {
     }
     const actualContainer = diagramContainer.current
     import('./markmap-loader').then(({ markmapLoader }) => {
-      const svg: SVGSVGElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg')
-      svg.setAttribute('width', '100%')
-      actualContainer.querySelectorAll('svg').forEach(child => child.remove())
-      actualContainer.appendChild(svg)
-      markmapLoader(svg, code)
+      try {
+        const svg: SVGSVGElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg')
+        svg.setAttribute('width', '100%')
+        actualContainer.querySelectorAll('svg').forEach(child => child.remove())
+        actualContainer.appendChild(svg)
+        markmapLoader(svg, code)
+      } catch(error) {
+       // console.error(error)
+      }
```

Comment:
Did we actually need this or could it be removed again? Then again maybe keep it, but uncomment the `console.error`.

Reply:
forgot to readd it. fixed